### PR TITLE
Fix Newsletter HTML issues

### PR DESCRIPTION
**Pull request by dhruvb14**
Fixed newsletter rendering `<hr>` outside of html spec causing WebKit based email rendering to show a lot of hr tags at the top of the email.

### CODE REVIEW

File: Ombi.Services/Jobs/RecentlyAdded.cs [hunk 1]
```diff
@@ -200,16 +200,17 @@ private void Start(NewletterSettings newletterSettings, bool testEmail = false)
                 html = template.LoadTemplate(sb.ToString());
                 Log.Debug("Loaded the template");
             }
-            
 
 
-            Send(newletterSettings, html, plexSettings, testEmail);
+            string escapedHtml = new string(html.Where(c => !char.IsControl(c)).ToArray());
```
Comment (tidusjar) [inline Ombi.Services/Jobs/RecentlyAdded.cs]: Why are we checking if there is a control character here?

Comment (dhruvb14) [inline Ombi.Services/Jobs/RecentlyAdded.cs]: For some reason when i was running through XHTML strict compliance checks it warned about the control character, browser ignore them but its not fully compliant. I can remove the check if needed but i just added it at the end.

Comment (tidusjar) [inline Ombi.Services/Jobs/RecentlyAdded.cs]: OK that's fine :) I was just wondering if there was something I was missing 😄

**Comment by tidusjar** [comment]
Excellent! Thank you very much for this!

File: Ombi.Services/Jobs/RecentlyAdded.cs (hunk 1 above)
Comment (dhruvb14) [inline Ombi.Services/Jobs/RecentlyAdded.cs]: I plan on adding a feature on the Newsletter tab allowing you to email everyone on your Plex updates for downtime and what not with a custom message section. Also do you have a backlog of features? I'm looking for a new project to start committing to and get my C# chops back up and build my Github profile out a bit :-)

Comment (tidusjar) [inline Ombi.Services/Jobs/RecentlyAdded.cs]: Hey @dhruvb14 jump on gitter! http://gitter.im/tidusjar/ombi

**Pull request merged**